examples/motion/proxy/motion.qi.idl

```diff
@@ -0,0 +1,112 @@
+package unknown
```

Comment:
please remove the methods not used in the example (maybe we can keep wakeUp(), rest(), and a few basic)
i like your idea of having an idl directory, let's do this in the future.